recipes-support/swupdate/swupdate_%.bbappend

```diff
@@ -5,4 +5,5 @@ FILESEXTRAPATHS:append := "${THISDIR}/${PN}:"
 
 PACKAGECONFIG_CONFARGS = ""
 
+
```

Comment:
nit: spurious newline -- remove this change from the change set?

Reply:
Addressed. Please re-review when able.